config/environments/production.rb

```diff
@@ -51,16 +60,11 @@
   # Don't log any deprecations.
   config.active_support.report_deprecations = false
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
-
-  # Use a different logger for distributed setups.
-  # require "syslog/logger"
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new "app-name")
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Comment:
Are we not using RAILS_LOG_TO_STDOUT env var in forms-deploy? if not then we should remove it

Reply:
[Seems that as 7.1 sets it to output to stdout by default](https://github.com/alphagov/forms-product-page/pull/88/files/527e086ac9780ed208ac73ff343693957d989ae1..a422830e0658f61f6ea643a83c7a846a4a649dc3#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eR40) so I'll remove from forms-deploy

Reply:
https://github.com/alphagov/forms-deploy/pull/395